ansys/dpf/core/fields_container.py

```diff
@@ -517,6 +517,11 @@ def animate(self, save_as=None, deform_by=None, scale_factor=1.0, **kwargs):
         # Define the field extraction using the fields_container and indices
         extract_field_op = dpf.core.operators.utility.extract_field(self)
 
+        loop_over = self.get_time_scoping()
+        frequencies = self.time_freq_support.time_frequencies
```

Comment:
@PProfizi I don't understand this changes, it means that we can't animate if we don't have a time freq support?

Reply:
@cbellot000 That was already the case, I just moved this up. Right now the animation is based on the TimeFreqSupport, yes.

Reply:
to add in the US to handle having no time freq support and to be able to create an animator with a mesh + a fields container